receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -626,6 +626,27 @@ func (s *MongoDBAtlasClient) GetClusters(ctx context.Context, groupID string) ([
 	return clusters, nil
 }
 
+// GetAlerts returns the alerts specified for the set projects
+func (s *MongoDBAtlasClient) GetAlerts(ctx context.Context, groupID string) ([]mongodbatlas.Alert, error) {
+	pageNum := 1
+	alertsResult := []mongodbatlas.Alert{}
+	for {
+		lo := mongodbatlas.ListOptions{PageNum: pageNum}
+		options := mongodbatlas.AlertsListOptions{ListOptions: lo}
+		alerts, response, err := s.client.Alerts.List(ctx, groupID, &options)
+		err = checkMongoDBClientErr(err, response)
+		if err != nil {
+			return nil, err
+		}
+		alertsResult = append(alertsResult, alerts.Results...)
+		if !hasNext(alerts.Links) {
+			break
```

Comment:
We need to limit the paging somehow. It's not reasonable to read all alerts every time.

Reply:
I'm resolving this based on the addition of the `max_alerts` setting.